src/api/todos.ts

```diff
@@ -5,4 +5,14 @@ export const getTodos = (userId: number) => {
   return client.get<Todo[]>(`/todos?userId=${userId}`);
 };
 
-// Add more methods here
+export const postTodo = (data: any) => {
```

Comment:
Don't use any for types. Add the correct type here

Reply:
fixed